nexus_constructor/nexus/nexus_wrapper.py

```diff
@@ -196,6 +196,11 @@ def set_field_value(
         :param dtype: Type of the value (Use numpy types)
         :return: The dataset.
         """
+
+        if isinstance(value, h5py.SoftLink):
+            group[name] = value
+            return value
```

Comment:
Is this returning a `h5py.SoftLink` when it should be returning an `h5py.Dataset`? (Should it `return group[name]`?)

Reply:
yes, it should.